src/tools/toolstate.toml

```diff
@@ -26,7 +26,7 @@
 miri = "Broken"
 
 # ping @Manishearth @llogiq @mcarton @oli-obk
-clippy = "Broken"
+clippy = "Building"
```

Comment:
This seems to break CI:
```
[00:02:42] failed to parse TOML configuration 'toolstate.toml': unknown variant `Building`, expected one of `Compiling`, `Testing`, `Broken` for key `clippy`

[00:02:42] failed to run: /checkout/obj/build/bootstrap/debug/bootstrap build nonexistent/path/to/trigger/cargo/metadata

[00:02:42] Build completed unsuccessfully in 0:01:16

[00:02:42] Makefile:79: recipe for target 'prepare' failed

[00:02:42] make: *** [prepare] Error 1

[00:02:42] Command failed. Attempt 2/5:

[00:02:42]     Finished dev [unoptimized] target(s) in 0.0 secs
```

Reply:
Darn it... last minute changes are never a good idea... sorry about that. I also fixed the wrong docs on the top of the file